projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot/sync-pilot.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="user$ | async as user">
+  <ng-container *ngIf="component$ | async as component">
+    <h3 *ngIf="!agent" class="table-header-title mb-0">
+      {{ 'fscommon.syncPilot.needHelp' | cxTranslate }}
+    </h3>
+    <div class="link" (click)="establishConnection(user, component, agent)">
+      <span *ngIf="agent; else noSpecificAgent" class="{{ agentIcon }}"></span>
```

Comment:
Shouldn't we render cx-icon here based on ICON_TYPE (or FS_ICON_TYPE) instead of passing this enum as a class?

Reply:
No big difference, but I could try with `cx-icon`.

Reply:
Somehow with cx-icon won't reflect the changes in the type, even though on `ngOnChanges` type is changed on the UI there are no changes.

Reply:
The result is maybe the same but Spartacus has created this feature for a [reason](https://sap.github.io/spartacus-docs/icon-library/#icon-component). We should try to use it and refactor the code where ever possible.